src/main/java/me/zeroeightsix/kami/module/modules/chat/AutoExcuse.kt

```diff
@@ -0,0 +1,59 @@
+package me.zeroeightsix.kami.module.modules.chat
+
+import me.zero.alpine.listener.EventHandler
+import me.zero.alpine.listener.EventHook
+import me.zero.alpine.listener.Listener
+import me.zeroeightsix.kami.event.events.GuiScreenEvent.Displayed
+import me.zeroeightsix.kami.module.Module
+import me.zeroeightsix.kami.setting.Setting
+import me.zeroeightsix.kami.setting.Settings
+import me.zeroeightsix.kami.util.MessageSendHelper
+import net.minecraft.client.gui.GuiGameOver
+import java.util.*
+
+/**
+ * @author sourTaste000
+ * @since 7/8/2020
+ * most :smoothbrain: code ever
+ */
+@Module.Info(
+        name = "AutoExcuse",
+        description = "Makes an excuse for you when you die",
+        category = Module.Category.CHAT
+)
+class AutoExcuse : Module() {
+
+    private val rand = Random()
+
+    private val modeSetting : Setting<Mode> = register(Settings.e("Mode", Mode.CRYSTAL))
+
+    @EventHandler
+    var listener = Listener(EventHook { event: Displayed ->
+        if (event.screen is GuiGameOver) {
+            //can't send message when player died
+            mc.player.respawnPlayer()
```

Comment:
won't this run more then once? getting rid of the GUI screen isn't instant with respawnPlayer()